### PR TITLE
Another Energy Gun Update

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request

- Adds a variable to set the muzzleflash to any color.
- Added dedicated sprites for laser bolts
- Made tesla rifle into a proper TGMC weapon rather than a generic "energy gun"

## Why It's Good For The Game
i am too tired to answer why okay

## Changelog
:cl:
add: Added new dedicated sprites for the laser bolts.
tweak: Changed the colors for the laser bolts.
tweak: Changed the tesla energy gun into a proper weapon.
tweak: Changed the colors of the muzzleflash for the energy weapons.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -7. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/guns/energy.dm [hunk 1]
```diff
@@ -135,23 +136,18 @@
...
-	aim_slowdown = 0.75
-	wield_delay = 1 SECONDS
-	gun_skill_category = GUN_SKILL_RIFLES
+	muzzle_flash_color = COLOR_TESLA_BLUE
 
 	charge_cost = 500
 	fire_delay = 4 SECONDS
-	accuracy_mult = 1.5
-	accuracy_mult_unwielded = 0.6
-	damage_falloff_mult = 0.5
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/energy.dm:154]: uhh?

File: code/modules/projectiles/guns/energy.dm [hunk 2]
```diff
@@ -135,23 +136,18 @@
 	damage_falloff_mult = 0.5
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/energy.dm:136]: The tesla gun is a child of the parent lasgun, so if no stats are inputted here, they'd fill up by the parent. And duplicating stats would add a little bloat.

Comment (TiviPlus) [reply code/modules/projectiles/guns/energy.dm:136]: okay if you say so

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-17). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**